src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -382,6 +384,17 @@ private static ILambdaExpression CreateBoundLambdaOperation(BoundLambda boundLam
             return new LazyLambdaExpression(signature, body, isInvalid, syntax, type, constantValue);
         }
 
+        private static ILocalFunctionStatement CreateBoundLocalFunctionStatementOperation(BoundLocalFunctionStatement boundLocalFunctionStatement)
+        {
+            IMethodSymbol localFunctionSymbol = boundLocalFunctionStatement.Symbol;
+            Lazy<IBlockStatement> body = new Lazy<IBlockStatement>(() => (IBlockStatement)Create(boundLocalFunctionStatement.Body));
+            bool isInvalid = boundLocalFunctionStatement.HasErrors;
+            SyntaxNode syntax = boundLocalFunctionStatement.Syntax;
+            ITypeSymbol type = null;
+            Optional<object> constantValue = default(Optional<object>);
```

Comment:
I have followed the general pattern used in this file. Heejae, you might want to take care of this when we checkin the generator.

---
In reply to: [121589002](https://github.com/dotnet/roslyn/pull/20177#discussion_r121589002) [](ancestors = 121589002)